mods/ra/maps/allies-06a/map.yaml

```diff
@@ -0,0 +1,650 @@
+MapFormat: 11
+
+RequiresMod: ra
+
+Title: 06a: Iron Curtain research center
+# Infiltrate and destroy Iron Curtain research center
```

Comment:
How many other long mission titles does RA have?  If this is a rare case then shortening the names is sufficient IMO.